go/4_auth-aws.go

```diff
@@ -66,7 +51,8 @@ func getSecretWithAWSAuthIAM() (string, error) {
 		return "", fmt.Errorf("data type assertion failed: %T %#v", secret.Data["data"], secret.Data["data"])
 	}
 
-	// data map can contain more than one key-value pair, in this case we're just grabbing one of them
+	// data map can contain more than one key-value pair,
+	// in this case we're just grabbing one of them
 	key := "password"
```

Comment:
Is `key` a separate variable to make it obvious that this is a map? Or can we consolidate this?

Reply:
I think I mostly just did it to prevent typing the same string multiple times since it's referenced two times in the error string just below that. Do you think that's good enough reason to keep it as a variable? Or is it impeding understanding?

Reply:
oh I see that now! Got it. I was thinking more for consistency sake. But that makes sense.